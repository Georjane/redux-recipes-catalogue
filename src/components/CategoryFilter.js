function CategoryFilter() {
  return (
    <div className="CategoryFilter">
      CategoryFilter
    </div>
  );
}

export default CategoryFilter;
