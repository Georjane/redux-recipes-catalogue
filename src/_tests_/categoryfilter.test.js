import React from 'react';
import renderer from 'react-test-renderer';
import { expect } from '@jest/globals';
import CategoryFilter from '../components/CategoryFilter';

it('renders CategoryFilter component', () => {
  const tree = renderer.create(<CategoryFilter handleFilterChange={() => {}} />);
  expect(tree).toMatchSnapshot();
});
