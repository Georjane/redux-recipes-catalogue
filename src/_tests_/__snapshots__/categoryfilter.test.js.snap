// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CategoryFilter component 1`] = `
<div
  className="filtercat"
>
  <p>
    <label
      htmlFor="cat"
    >
      Choose
      <select
        className="form-control"
        id="cat"
        onChange={[Function]}
      >
        <option>
          CATEGORIES
        </option>
        <option>
          All
        </option>
        <option>
          Beef
        </option>
        <option>
          Chicken
        </option>
        <option>
          Dessert
        </option>
        <option>
          Pasta
        </option>
        <option>
          Pork
        </option>
        <option>
          Seafood
        </option>
        <option>
          Vegan
        </option>
        <option>
          Breakfast
        </option>
      </select>
    </label>
  </p>
</div>
`;
